go/private/actions/archive.bzl

```diff
@@ -42,18 +42,18 @@ def _go_archive_aspect_impl(target, ctx):
   direct = []
   for dep in ctx.rule.attr.deps:
     direct.append(get_archive(dep))
-  for dep in ctx.rule.attr.embed:
-    direct.extend(get_archive(dep).direct)
+  for embed in ctx.rule.attr.embed:
+    direct.extend(get_archive(embed).direct)
```

Comment:
I'm getting a bazel analysis failure on this line now if I combine `bazel test --features=race` with `race = "off"` on a rule:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/5505/pull-test-infra-bazel/10159/?log#log
```
W1116 23:38:33.335] ERROR: /workspace/k8s.io/test-infra/mungegithub/mungers/BUILD:9:1: in @io_bazel_rules_go//go/private:actions/archive.bzl%go_archive_aspect[pure="auto",static="auto",msan="auto",race="off"] aspect on go_library rule //mungegithub/mungers:go_default_test~library~: 
W1116 23:38:33.336] Traceback (most recent call last):
W1116 23:38:33.336] 	File "/workspace/k8s.io/test-infra/mungegithub/mungers/BUILD", line 9
W1116 23:38:33.337] 		@io_bazel_rules_go//go/private:actions/archive.bzl%go_archive_aspect(...)
W1116 23:38:33.337] 	File "/root/.cache/bazel/_bazel_root/27a95fe2fc68aa1195ddf1a4bbaaea62/external/io_bazel_rules_go/go/private/actions/archive.bzl", line 46, in _go_archive_aspect_impl
W1116 23:38:33.338] 		direct.extend(get_archive(embed).direct)
W1116 23:38:33.338] argument 'items' has type 'depset', but should be 'sequence'
W1116 23:38:33.339] in call to builtin method list.extend(items).
W1116 23:38:33.494] ERROR: Analysis of target '//mungegithub/mungers:go_default_test' failed; build aborted: Analysis of target '//mungegithub/mungers:go_default_test~library~' failed; build aborted.
```

Reply:
Sorry for the break. It looks like #1029 will fix this.